docs/marketplace/add_to_marketplace/marketplace_items_accessibility.md

```diff
@@ -0,0 +1,43 @@
+---
+id: marketplace_items_accessibility
+title: Make your Marketplace items available to other Companies
+sidebar_label: Make your Marketplace items available to other Companies
+---
+
+You can contribute to the Mia-Platform Marketplace with the Marketplace items of your Company, by deciding to make them accessible also to other Companies. This section will guide you on how to do so, offering two main methods:
+
+* (recommended) Use the [CMS](/business_suite/guide_cms.md) to make your Marketplace item accessible to other Companies.
+* Open an issue on Mia-Platform [Github community page](https://github.com/mia-platform/community).
+
+Both methods will be explained in the following paragraphs.
+
+## Use the CMS to make your Marketplace item accessible to others
+
+To make a Marketplace item of your Company accessible to other Companies, you first need to create it in the CMS. Follow the instructions on [this page](/marketplace/add_to_marketplace/contributing_overview.md#how-to-configure-a-new-component) to do so.  
+Once you have created your Marketplace item in the CMS, you can make it accessible to other companies by editing its `visibility` property. Specifically, set the `allTenants` property inside the `visibility` object to "true".  
+After editing, the visibility property should look like this:
+
+```json
+{
+    "allTenants": true
+}
+```
+
+If the repository where your Marketplace item is located is freely accessibile(open source), then you have completed the process, and your marketplace item can now be accessed and used by other Companies!
+
+Otherwise, in order to grant access to others Companies, you should provide them with a **Provider** that has the required permissions to access your Marketplace item repository and make it accessible to other companies.
```

Comment:
```suggestion
Otherwise, in order to grant access to others Companies, you should provide them with a **Provider** that has the required permissions to access your Marketplace item repository and make it accessible to other Companies.
```